docs/topics/gradle.md

```diff
@@ -624,6 +614,188 @@ The complete list of options for the Gradle tasks is the following:
 | `target` | Generate JS files for specific ECMA version | "v5" | "v5" |
 | `typedArrays` | Translate primitive arrays to JS typed arrays |  | true |
 
+### Attributes specific for JVM
+
+| Name | Description | Possible values |Default value |
+|------|-------------|-----------------|--------------|
+| `javaParameters` | Generate metadata for Java 1.8 reflection on method parameters |  | false |
+| `jdkHome` | Include a custom JDK from the specified location into the classpath instead of the default JAVA_HOME. Direct setting is deprecated sine 1.5.30, use [other ways to set this option](#setting-custom-jdk-home).  |  |  |
```

Comment:
```suggestion
| `jdkHome` | Include a custom JDK from the specified location into the classpath instead of the default JAVA_HOME. Direct setting is deprecated sinсe 1.5.30, use [other ways to set this option](#setting-custom-jdk-home).  |  |  |
```